swap_meet/vendor.py

```diff
@@ -1,2 +1,82 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory
+        if self.inventory == None:
+            self.inventory = []
+    
+    def add(self, item):
+        if self.inventory == None:
+            self.inventory = []
+            self.inventory.append(item)
+        else:
+            self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
Because the `if` block always returns, the `else` is unneeded. You could instead do:

```
    def remove(self, item):
        if item in self.inventory:
            self.inventory.remove(item)
            return item
        return False
```